src/encoded/static/components/app.js

```diff
@@ -1742,13 +1742,7 @@ class BodyElement extends React.PureComponent {
      */
     getViewportDimensions(){
         if (isServerSide()) return;
-
-        var scrollElem      = (window.document && window.document.scrollingElement) || null,
-            windowWidth     = (scrollElem && (scrollElem.clientWidth || scrollElem.offsetWidth)) || window.innerWidth,
-            windowHeight    = (scrollElem && (scrollElem.clientHeight || scrollElem.offsetHeight)) || window.innerHeight;
-            //documentHeight  = (widthElem && widthElem.scrollHeight); Not relevant re: resizing.
-
-        return { windowWidth, windowHeight };
+        return { 'windowWidth' : window.innerWidth , 'windowHeight' : window.innerHeight };
```

Comment:
Changed from document.scrollingElement (usually alias of document.body or document html element) to window inner width to align with CSS window-size-based breakpoints.

Downside is some elements/things might appear a little bit wider-than-available-space now on _small_ screens or when full screen because the scrollbar width is included in `window.innerWidth`.

However, (a) small screen sizes occur mostly on mobile where the scrollbar is not rendered and (b) when full-screen, scrollbar is forcibly hidden and window is not usually scrollable (exception: BrowseView, where doesn't matter) so this seemed acceptable.